infrastructure/pocket-event-bridge/src/events-schema/ForgotPasswordRequestEvent.ts

```diff
@@ -17,10 +17,10 @@ export class ForgotPasswordRequestEvent extends Construct {
     private name: string,
   ) {
     super(scope, name);
-    this.createPremiumRenewalUpcomingEvent();
+    this.createForgotPasswordEvent();
   }
 
-  private createPremiumRenewalUpcomingEvent() {
+  private createForgotPasswordEvent() {
     const schemaProps: SchemasSchemaConfig = {
       name: this.forgotPasswordRequest,
```

Comment:
Should this match the value in eventConfig.ts?   (`ForgotPassword` vs. `Forgot-Password-Request`)

not sure importance since it's forwarded on detailType

Reply:
Updated for consistency